parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -168,21 +168,7 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
-    {
-      "live-restore": true,
-      "log-driver": "json-file",
-      "log-opts":  {
-         "max-size": "50m",
-         "max-file": "5"
-      }{{if IsNSeriesSKU .}}
-      ,"default-runtime": "nvidia",
-      "runtimes": {
-         "nvidia": {
-             "path": "/usr/bin/nvidia-container-runtime",
-             "runtimeArgs": []
-        }
-      }{{end}}
-    }
+{{IndentString (GetDockerConfig (IsNSeriesSKU .VMSize)) 4}}
```

Comment:
is there a way to toggle this on/off? What if the changes doesnt work as expect?

Reply:
The default change implemented by the PR is a no-op. In RP, the implementation would be as a toggle that sets the config value to e.g. "/mnt/docker". 

I don't know of another way to toggle in yaml directly.

Reply:
got you. thanks.